cpp/log/test.cpp

```diff
@@ -61,4 +68,76 @@ void test() {
 	t1.join();
 }
 
+/**
+ * Creates a FileSink to also write log Messages
+ * into /tmp/openage-log-test.
+ * Only Messages with a level equals or higher
+ * than level::err will be printed in that file.
+ */
+void file_log_sink () {
+	const char *filelog_name = "/tmp/openage-log-test";
```

Comment:
hard-coding the filename to `/tmp` is sub-optimal; this might not work on OSX and will not work on Win32. Use `mkstemp` instead.